app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,19 @@ def show
     @user = @post.author
   end
 
+  def destroy
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find_by(id: params[:id])
```

Comment:
- [ ] Please make sure the app uses `CanCanCan` for Post authorization in Post Controller. 
  - You can consider using the below code:
  
```ruby
 class PostsController < ApplicationController
    load_and_authorize_resource
    ...
  end
```